README.md

```diff
@@ -166,7 +167,20 @@ This is an overview of the steps that the `trex run10x`/ `trex smartseq3` comman
 - Reads do not get grouped into molecules and their cloneIDs are not collapsed 
   and error-corrected into consensus sequences
 
-## Input files
+
+## Input parameters
+
+### Jaccard index threshold
+
+The Jaccard index measures the similarity of two sample sets, in this case 
+the similarity of two sets of cloneIDs. It is calculated by dividing
+the number of overlapping, unique cloneIDs between cell A and B by the total 
+number of unique cloneIDs in cell A and B. An index of `0` indicates no 
```

Comment:
```suggestion
number of unique cloneIDs in cell A and B. An index of 0.0 indicates no 
```
(No backticks because this doesn’t need to be formatted as code. And write 0.0 to emphasize that the Jaccard index is a floating-point value.)